exercises/sum.cpp

```diff
@@ -6,3 +6,24 @@
   Insert the second number: 2
   Sum: 3
 */
+
+#include <iostream>
+using namespace std;
+
+int main() {
+    int num1, num2, sum;
+
+    // prendo in input i due numeri
+    cout << "Insert the first number: ";
+    cin >> num1;
+    cout << "Insert the second number: ";
+    cin >> num2;
+
+    // sommo i due numeri
+    sum = num1 + num2;
+
+    // stampo la somma
+    cout << "Sum: " << sum << endl;
+
+    return 0;
```

Comment:
potresti sostituire 0 con EXIT_SUCCESS in modo da migliorare la leggibilità del codice
```c++
#define EXIT_SUCCESS 0
...
return EXIT_SUCCESS;
```